feat: `max-length` config (#194)
Nutlope/aicommits: README.md

aicommits config set timeout=20000 # 20s
```

#### length

> The maximum character length to be generated for the commit message.

Reviewer: ```suggestion
The maximum character length of the generated commit message.
```

aicommits config set timeout=20000 # 20s
```

#### length

> The maximum number of tokens to be generated for the commit message.

Reviewer: ```suggestion
The maximum character length for the commit message.
```

> Default: `20` (80 characters)

Reviewer: For ease of use, this should accept character length, and we can convert that into tokens for them.

Author: How do you suggest to convert character length (int) to convert to number of tokens?

Reviewer: Ah true. Yeah using TikToken for a baseline is fine. I think maybe add 5~10 characters for padding.

We should also include in the prompt to limit the message to under x characters.

Author: We can also use this logic I guess `(Number of chars + 5) / 4`. This will pretty much result the same as TikToken to get the number of tokens.

https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them
